include/vapor/ConvexHull.h

```diff
@@ -0,0 +1,152 @@
+// A C++ program to find convex hull of a set of points. Refer
+// https://www.geeksforgeeks.org/orientation-3-ordered-points/
+// for explanation of orientation()
+#include <iostream>
+#include <stack>
+#include <stdlib.h>
+using namespace std;
+
+/*struct Point
+{
+    double x, y;
+};*/
+
+// A global point needed for  sorting points with reference
+// to  the first point Used in compare function of qsort()
+glm::vec2 p0;
```

Comment:
Good point.  I was hoping to be as hands-off as possible with this code since it's not mine, but it looks like it should be broken into a source file with namespace.